labels/de-DE.json

```diff
@@ -86,6 +86,7 @@
   "creditAmount": "Kreditbetrag",
   "creditCard": "Kreditkarte",
   "creditDate": "Kredit Datum",
+  "createQuote": "Angebot erstellen",
```

Comment:
This is in the wrong spot. Please place this in the right spot alphabetically. It should be after `createAccount` I think if capitals sort before lowercase. Otherwise after createdDate. You can check this online with an alphasort.

Reply:
Added at correct location